chart/values.yaml

```diff
@@ -2277,8 +2277,6 @@ cleanup:
 # Not recommended for production
 postgresql:
   enabled: true
-  image:
-    tag: "11"
```

Comment:
I think it is better use specific **major** Postgres version here, rather than use default from bitnami/postgres subchart.

```yaml
  image:
    tag: "16"
```

`16` - referenced to the latest minor version of [`bitnami/postgresql`](https://hub.docker.com/r/bitnami/postgresql/tags) of Postgres 16, it is in opposite of `16.1.0-debian-11-r15` which are referenced to the specific major.minor version

Reply:
Thank you for the review.
There is a good practice to use specific version rather than any sort of latest version as far as I know. Specific version provides better guarantees for deployment to be reproduceable and clear expectations on what exactly should be deployed:
1) Two `bitnami/postgresql` deployments with `{{ .Values.image.tag }} = 16` might run different database versions. For instance, today it deploys  `16.1.0-debian-11-r15`, but few days ago `16.1.0-debian-11-r13` was deployed with the same `values.yaml` file.
2) The deployment with `{{ .Values.image.tag }} = 16` not necessarily runs latest 16 version. For instance, because the chart was deployed a week ago when `16.1.0-debian-11-r15` didn't exist.
3) Fresh deployment with `{{ .Values.image.tag }} = 16` not necessarily runs latest 16 version, because docker image with tag 16 might be presented in host from previous or parallel deployments.

Also, default value from `bitnami/postgres` was used before the https://github.com/apache/airflow/pull/29207 where 11 version was pinned to resolve the upgrade issue temporary.

Reply:
Differences between minor version of Postgres is a bug fixes include security one, data files are binary compatible between same major version.

So if only pin to the specific version then all users who doesn't care about specify version they would be get one of the first release of 16 Postgres, without fixes.

Reply:
BTW, this only my suggestion, but not strong opinion

Reply:
I see your point. Let's see what others think.

Reply:
I think it's fine.. Postrgres chart is only used for testing/dev so we can get the default one.